packages/codec-components/src/react/components/codec/format.errors.deployed-function-in-constructor-error.tsx

```diff
@@ -2,12 +2,18 @@ import React from "react";
 import type { Format } from "@truffle/codec";
 import { createCodecComponent } from "../../utils/create-codec-component";
 import { CodecError } from "../common/codec-error";
+import { FunctionInternalRawInfoPcPair } from "./format.values.function-internal-raw-info-pc-pair";
 
 export const { DeployedFunctionInConstructorError } = createCodecComponent(
   "DeployedFunctionInConstructorError",
-  ({ kind, context }: Format.Errors.DeployedFunctionInConstructorError) => (
+  ({
+    kind,
+    context,
+    rawInformation
+  }: Format.Errors.DeployedFunctionInConstructorError) => (
     <CodecError kind={kind}>
       Deployed-style pointer in {context.typeName} constructor
+      <FunctionInternalRawInfoPcPair data={rawInformation} />
```

Comment:
There should probably be a whitespace before this, `&nbsp;` or `{" "}`.

Reply:
I see!  Things I will have to keep in mind...